app/views/layouts/_dmponline_footer.html.erb

```diff
@@ -10,14 +10,14 @@
       <% end %>
 
    			</p>
-       	
-    </div>	
+
+    </div>
    	<div class="right_side_footer">
-   		<br/> <br/> <a href="https://library.ualberta.ca/about-us/policies/privacy-policy">Web Site Privacy Policy</a> | <%= link_to t('helpers.terms'), terms_path %>
+        <br/> <br/> <a href="https://library.ualberta.ca/about-us/policies/privacy-policy">Web Site Privacy Policy</a> | <a href='terms'>Terms and conditions</a>
```

Comment:
Sorry @astrilet The url to terms need to be replaced by this link https://portagenetwork.ca/portage-dmp-assistant-terms-use/